static/js/tools/timeline/chart.tsx

```diff
@@ -173,6 +182,13 @@ class Chart extends Component<ChartPropsType, ChartStateType> {
     this.loadRawData();
     this.resizeObserver = new ResizeObserver(this.handleWindowResize);
     this.resizeObserver.observe(this.svgContainer.current);
+    // Triggered when the component is mounted and send data to google analytics.
+    window &&
+      window.gtag &&
+      window.gtag("event", "tool_chart_plot", {
+        place_dcid: Object.keys(this.props.placeNames),
```

Comment:
how does the list get interpreted by GA? will [a, b] be treated separately from [a] and [b] (or would we be able to get stats for 'a' in the first example)?

Reply:
GA will change the list to a comma-separated string. For example,  if we have [geoId/06, geoId/0667000], in the GA report, we will get "geoId/06, geoId/0667000". We could only see a combination of them. But we can see the statistics for every single place from the event "tool_place_add".

Reply:
> 

Done. Define constants in a new file for events, parameters and values.